contracts/handlers/GenericHandler.sol

```diff
@@ -112,6 +115,14 @@ contract GenericHandler is IDepositHandler, ERC20Safe {
             )
         }
 
+        address contractAddress = _resourceIDToContractAddress[resourceID];
+        require(_contractWhitelist[contractAddress], "provided contractAddress is not whitelisted");
+
+        if (_contractAddressToDepositFunctionSignature[contractAddress] != bytes4(0)) {
+            (bool success, bytes memory returnedData) = contractAddress.call(metaData);
```

Comment:
Currently `returnedData`  is not used because I'm not sure how to implement a generic way to handle this (same for `executeDeposit` method)

Related #95

Reply:
I think it can be discarded, the requirement for any deposit check (implemented in the target contract) should be: revert if not allowed to do this action, not much use for returned values

Reply:
what if we exclusively just emit the return data?

Reply:
For what purpose?